docs/framework/resources/creating-satellite-assemblies-for-desktop-apps.md

```diff
@@ -89,8 +89,13 @@ The following is a simple "Hello world" example that displays a message box cont
 
 2. To indicate that English (en) is the application's default culture, add the following <xref:System.Resources.NeutralResourcesLanguageAttribute?displayProperty=nameWithType> attribute to the application's AssemblyInfo file or to the main source code file that will be compiled into the application's main assembly.
 
-    [!code-csharp[Conceptual.Resources.Locating#2](~/samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs#2)]
-    [!code-vb[Conceptual.Resources.Locating#2](~/samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.resources.locating/vb/assemblyinfo.vb#2)]  
+    ```csharp
+    [assembly: NeutralResourcesLanguageAttribute("en")]
```

Comment:
**Unrelated to the PR**

https://github.com/dotnet/sdk/blob/94a3f2856cb09e66ee7472820b4e26fb576b4686/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets#L83

This example shows the alternative way to do this in the csproj, instead of AssemblyInfo.

If this works with SDK-style projects for .NET Framework, the article should be updated to reflect that.

Also, I think satellite assemblies might not be .NET Framework-specific. So, the article may be moved to be more general.

Reply:
Yes, it should probably be moved out of the framework directory. Can you log an issue and perhaps add it to this project? https://github.com/dotnet/docs/projects/109